### ISSUE TITLE
new metric: Popularity Metric for Living Wage

### ISSUE THREAD

**Issue by GeorgLink** [chaoss/wg-value#104]
@hickmanjv proposed this metric in chaoss/wg-value#83.

We will revise the metric in this Google Doc: https://docs.google.com/document/d/1uMYo80oeA9Q_ky_llFunHfeUNTxAgT2tW5kWJcu4GIA/edit?usp=sharing

Once the revision is done, we'll open a new PR.

**Comment by mbbroberg** [comment]
Hey @hickmanjv, thanks again for proposing this metric and your patience while we wait to discuss it until the metric release completed. We took the time today, and here are some of the key takeaways. 

The CHAOSS maintainers believe that Popularity and Living Wage may overlap in atomic parts, but offer clearer insights as separate metrics. The former is actually due for an update, captured in #85, and we started to use your metric to update that one. Please jump into that discussion if you're interested. 

Regarding Living Wage, we have a related metric already in [Job Opportunity](https://github.com/chaoss/wg-value/blob/master/focus-areas/individual-value/job-opportunities.md), which may warrant updates if you have thoughts to add there. Thinking of Living Wage in and of itself, it's not clear what atomic metrics can be aggregated in a way to calculate a wage can be earned through learning a given technology. I'm left with a question of whether we have the right atomic metrics available -- is there a salary range by topic metric that could be created? I believe approaching this from a programmatic and smaller scope like this question could get you toward your goals there. 

To address the other part, the BOSS index is a closed-source business value measurement for open source projects. To frame it as a question, it would be "is this open source project worth financially investing in?" That could also be a new direction to take a metric. 

In short, there are a lot of ways to continue this exploration. First up from the project point of view is #85, though we welcome other efforts through updating existing metrics or offering other new ones. 

We'll leave this open until our next meeting in 2 weeks. Feel free to join in if you can or comment here.
